doc/changelog.rst

```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+Version 2.6.3 (2024-03-05)
+---------------------------
+
+* Support Free-threading CPython (PEP-703) by disabling psutil related features.
```

Comment:
Please add a link to the issue.

Reply:
Which link? PEP 703?

Reply:
The current blocker is https://github.com/pypa/packaging/issues/755, but there's ~3 more steps needed after that too (update packaging in pip, release pip, update pip in CPython).

Reply:
The issue describing why pyperf cannot use psutil on a Python Free Threading build.

Reply:
Done.